server/http_test.go

```diff
@@ -22,6 +22,7 @@ import (
 const TestDataLongSubscriptionName = `aaaaaaaaaabbbbbbbbbbccccccccccddddddddddaaaaaaaaaabbbbbbbbbbccccccccccddddddddddaaaaaaaaaabbbbbbbbbbccccccccccddddddddddaaaaaaaaaabbbbbbbbbbccccccccccddddddddddaaaaaaaaaabbbbbbbbbbccccccccccddddddddddaaaaaaaaaabbbbbbbbbbccccccccccddddddddddaaaaaaaaaabbbbbbbbbbccccccccccdddddddddd`
 
 var testSubKey = keyWithInstanceID(mockInstance1URL, JiraSubscriptionsKey)
+var testTemplateKey = keyWithInstanceID(mockInstance1URL, templateKey)
```

Comment:
`mockInstace1URL` is this "1" in the name intentional?

Reply:
yes they have created mocks for instance store and give that this name and I have just used that